flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -97,6 +97,21 @@
 				" different TaskManagers need different values for this option, usually it is specified in an" +
 				" additional non-shared TaskManager-specific config file.");
 
+	/**
```

Comment:
Ok, I could revert this change. 
Just because it was in the network section, and after I migrated the network section out, it seems exist lonely.  :)